### ISSUE TITLE
[Bug]: camera "motion_detected" not working

### ISSUE THREAD

**Issue by scottmil** [bropat/ioBroker.eusec#263]
### Adapter version

0.9.1

### JS-Controller version

4.0.23

### Node version

17.9.0

### Operating System type

Other

### Operating system version

Raspberry Pi Raspian Buster

### Describe the bug

Camera "motion_detected" does not change value from "false" to "true" when motion detection is enabled on my Eufy Cam2C (T8142Z) cameras and Eufy Battery Doorbell (T8210B).  The Eufy iPhone app correctly records when motion is detected on the cameras.   

BTW: The adapter does correctly record changes to the "ringing" status when the doorbell button is pressed.

### To reproduce

1. Set eusec "motion_detection" to "true" by enabling camera motion detection using the Eufy iPhone app. (I can also enable motion detection using a custom Hubitat driver and device.)
2. Manually create a motion detected Eufy event by standing in front of a camera while simultaneously watching for change in state of the eusec "motion_detected" state.  (I also created a Blockly script to notify Hubitat of a change in state of the eusec "motion_detected". )
3. The "motion_detected" state never changes from "false" to "true" in ioBroker.  Hubitat is also never notified of a change in state.
4. Screenshots
<img width="1085" alt="Eufy Camera Motion" src="https://user-images.githubusercontent.com/10510570/176331530-659627e6-1ebf-44ea-951b-d6c299f53190.png">

<img width="649" alt="Script" src="https://user-images.githubusercontent.com/10510570/176331599-d91ce06a-5de6-478a-bd2c-39b7369a2f81.png">

**Comment by bropat** [comment]
@scottmil 

`motion_detected` is **only** based on push notifications (no other known method).
If you have deactivated push notifications, you will never receive a motion detection.

**Comment by scottmil** [comment]
Per the attached iPhone Eufy app screenshot for my Eufy Battery Doorbell, it appears that push notifications are activated.  I receive motion detection notifications on my iPhone, but not in ioBroker.  Is there somewhere else I need to configure push notifications?   

In ioBroker, I also set “notification_motion” from false to true for my Eufy Doorbell, but it had no effect.

As mentioned previously, ioBroker does change the value of the Doorbell “ringing” state when I press the doorbell button.

**Comment by scottmil** [comment]
04-July-2022 Update: 1. I discovered that Eufy will only generate notifications if BOTH "Video Recording" AND "Notification" are checked in the Eufy app Security Settings.  Checking only "Notification" did not generate a notification on either my iPhone or ioBroker.
2. After experimenting, I found that the eusec object "person_detected" was a more reliable monitor of motion detection compared to the "motion_detected" object.  I edited my Blockly script to notify Hubitat of changes in the "person_detected" state, and it appears to working consistently.

<img width="1534" alt="iobroker-person_detected" src="https://user-images.githubusercontent.com/10510570/177213430-d3ffcd43-cb17-41dc-b88a-0cff92732f08.png">

<img width="840" alt="Blockly-Hubitat" src="https://user-images.githubusercontent.com/10510570/177213625-3465d088-ee78-497b-a797-44779284e541.png">

<img width="893" alt="Hubitat-notified" src="https://user-images.githubusercontent.com/10510570/177213656-c1d06e47-760a-4079-bfd9-9a277fd988a7.png">
